ggdp/__init__.py

```diff
@@ -18,6 +20,7 @@
 all_assets = load_assets_from_modules([assets])
 
 resources = {
+    "covalentAPI": res.CovalentAPIResource(API_KEY=EnvVar("COVALENT_API_KEY")),
```

Comment:
```suggestion
    "covalent_api": res.CovalentAPIResource(API_KEY=EnvVar("COVALENT_API_KEY")),
```

Also curious if `EnvVar` works here. I had some issues with it a few months ago when they introduced it!

Reply:
Worked for me without problem, I also find it produces a nicer error message.

Reply:
Nice! We should update all the `os.getenv` to this as we encounter them!